cookbook/case_studies/feature_engineering/feast_integration/Feast_Flyte_Demo.ipynb

```diff
@@ -2,260 +2,406 @@
   },
   {
    "cell_type": "markdown",
-   "id": "adb68140",
+   "id": "07bd9e37",
    "metadata": {},
    "source": [
-    "## Step 3: Now wait for the execution to complete\n",
-    "It is possible to launch a sync execution and wait for it to complete, but since all the processes are completely detached (you can even close your laptop) and come back to it later, we will show how to sync the execution back."
+    "### Step 3: Wait for the execution to complete \n",
+    "\n",
+    "It is possible to launch a sync execution and wait for it to complete, but since all the processes are completely detached (you can even close your laptop and come back to it later), we will show how to sync the execution back.\n",
+    "\n",
+    "**Side Note**\n",
+    "It is possible to fetch an existing execution or simply retrieve a started execution. Also, if you launch an execution with the same name, Flyte will respect that and not restart a new execution!"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "502711ef",
+   "execution_count": 5,
+   "id": "a8bd9614",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Execution f3fdc5c8c0e434011911 is in Phase - SUCCEEDED\n"
+     ]
+    }
+   ],
    "source": [
     "from flytekit.models.core.execution import WorkflowExecutionPhase\n",
-    "exe = remote.sync(exe)\n",
-    "print(f\"Execution {exe.id.name} is in Phase - {WorkflowExecutionPhase.enum_to_string(exe.closure.phase)}\")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "9fda8da9",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "exe.sync()"
+    "\n",
+    "synced_execution = remote.sync(execution)\n",
+    "print(f\"Execution {synced_execution.id.name} is in Phase - {WorkflowExecutionPhase.enum_to_string(synced_execution.closure.phase)}\")"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "4c2074cc",
+   "id": "65e5b181",
    "metadata": {},
    "source": [
-    "## Step 4: Lets sync data from this execution\n",
-    "\n",
-    "**Side Note**\n",
-    "It is possible to fetch an existing execution or simply retrieve a started execution. Also if you launch an execution with the same name, flyte will respect and not restart a new execution!\n",
+    "### Step 4: Retrieve output\n",
     "\n",
-    "To fetch an execution\n",
-    "```python\n",
-    "exe = remote.fetch_workflow_execution(name='f9f180a56e67b4c9781e')\n",
-    "exe = remote.sync(exe)\n",
-    "```"
+    "Let's fetch the workflow outputs."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "3e778b53",
+   "execution_count": 6,
+   "id": "ab24b1c0",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from feast_dataobjects import FeatureStore\n",
-    "fs = exe.raw_outputs.get('o0', FeatureStore)\n",
-    "model = exe.outputs['o1']"
+    "fs = synced_execution.outputs['o0']\n",
```

Comment:
Does this work? How can it get the right thing

Reply:
I didn't get you. Why shouldn't it?

Reply:
If you mean to ask if the output is a FeatureStore data class with all the methods within it accessible, then no, it isn't precisely so. Hence, what I am doing is, accessing the config and creating a FeatureStore out of it. 

```python
import os

from feast_workflow import predict, FEAST_FEATURES
from feast_dataobjects import FeatureStore

os.environ["FLYTE_AWS_ENDPOINT"] = os.environ["FEAST_S3_ENDPOINT_URL"] = "http://localhost:30084/"
os.environ["FLYTE_AWS_ACCESS_KEY_ID"] = os.environ["AWS_ACCESS_KEY_ID"] = "minio"
os.environ["FLYTE_AWS_SECRET_ACCESS_KEY"] = os.environ["AWS_SECRET_ACCESS_KEY"] = "miniostorage"

inference_point = FeatureStore(fs.config).get_online_features(FEAST_FEATURES, [{"Hospital Number": "533738"}])
inference_point
```

Does this not look okay? I tried running the code with `raw_outputs` and this is what I see:

```
AttributeError: 'FlyteWorkflowExecution' object has no attribute 'raw_outputs'
```

Reply:
Ohh ya I had hacked that method, I think there is a better method now

Reply:
@kumare3 we haven't merged that PR yet, so we can't get "raw_outputs" now.
https://github.com/flyteorg/flytekit/pull/675
I think that PR is ready to be merged after resolving the merge conflict.

Reply:
@kumare3, https://github.com/flyteorg/flytekit/pull/675 isn't merged yet, so I don't think we still have a better method. Am I correct?

Reply:
@pingsutw, can you please merge https://github.com/flyteorg/flytekit/pull/675? I think @kumare3 wants you to finalize, clean up the code, and merge the PR.

Reply:
@kumare3, code is working now. Can we merge this PR?